3.landing-page-node/views/scripts/landing-page.js

```diff
@@ -47,10 +48,27 @@ function ListenToRequest(){
         .catch(function (error) {
             console.log(error);
         });
-
+        
     });
+
```

Comment:
document.getElementById('xxx');
can resolve to undefined
alway do an if after statement like that to check if it found something
(for example
if (openChat) {
 // add event listener.. 
}

)